app/src/main/java/com/orange/ouds/app/ui/components/checkbox/CheckboxItemDemoState.kt

```diff
@@ -31,7 +31,7 @@ fun rememberCheckboxItemDemoState(
         ToggleableState.Off
     ), // only used for indeterminate checkbox item demo
     icon: Boolean = false,
-    divider: Boolean = true,
+    divider: Boolean = false,
```

Comment:
Should we also update the values in the preview parameter?

We currently have 3 parameters, first and second one use the default value (which is still `true` for the parameter) and the last one forces it to `false`.